app/src/main/java/ar/com/wolox/android/cookbook/graphQl/OrdersFragment.kt

```diff
@@ -0,0 +1,59 @@
+package ar.com.wolox.android.cookbook.graphQl
+
+import android.os.Bundle
+import android.view.View
+import androidx.recyclerview.widget.LinearLayoutManager
+import ar.com.wolox.android.cookbook.R
+import ar.com.wolox.android.cookbook.model.ModelOrder
+import ar.com.wolox.wolmo.core.fragment.WolmoFragment
+import kotlinx.android.synthetic.main.fragment_orders.*
+
+class OrdersFragment : WolmoFragment<OrdersPresenter>(), OrdersView {
+
+    private lateinit var ordersAdapter: OrdersAdapter
+    private lateinit var layoutManager: LinearLayoutManager
+
+    override fun init() {
+        presenter.init()
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        setUpRecyclerView()
+        setOnClickListeners()
+    }
+
+    override fun layout(): Int = R.layout.fragment_orders
+
+    override fun onOrdersLoaded(orders: List<ModelOrder>) {
+        requireActivity().runOnUiThread {
+            toggleProgress(false)
+            ordersAdapter.setOrders(orders)
+        }
+    }
+
+    override fun onError() {
+        TODO("not implemented") // To change body of created functions use File | Settings | File Templates.
+    }
+
+    private fun setUpRecyclerView() {
+        ordersAdapter = OrdersAdapter(requireContext())
+        layoutManager = LinearLayoutManager(requireContext())
+        vRecyclerView.apply {
+            adapter = ordersAdapter
+            layoutManager = layoutManager
+            isNestedScrollingEnabled = false
+        }
+    }
+
+    private fun setOnClickListeners() {
+        vMakeGraphQLRequest.setOnClickListener {
+            toggleProgress(true)
+            presenter.getOrders(1, 10)
```

Comment:
You can add this numbers, or at least the 10 into a companion object like const MAX something